advocacy_docs/partner_docs/CohesityBackupandRecover/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+Using the Cohesity Postgres Connector Agent, you can backup and restore EDB Postgres Advanced Server databases, as well as set specific backup policies for your databases and keep track of the backup and restore runs. Cohesity’s backup and recovery tool for Postgres provides peace of mind for database owners with its nontraditional approaches to backups.  The Cohesity Platform natively integrates with PostgreSQL to provide backup and recovery for PostgreSQL deployments. Cohesity’s Datasource Connector Agents allow for flexibility of customization, including being part of Cohesity’s data management platform, Helios.
+
+The Cohesity Platform provides a PostgreSQL connector that enables you to perform full backups, incremental (differential) backups, log backups, restores and point in time restores, all at the cluster level.
+
+The following are the core components for protecting EDB Postgres Advanced Server databases with Cohesity:
+
+- EDB Postgres Advanced Server
+- Cohesity Backup Appliance
+- Cohesity PostgreSQL connector
+- Cohesity Connector Agent
+
+
+The diagram below shows how you can integrate the Cohesity Platform with your EDB Postgres Advanced Server deployments using the Cohesity PostgreSQL connector and Connector Agent.
```

Comment:
"Platform" should be lowercase and "Connector" in "Cohesity PostgreSQL connector" should be uppercase.